### PR TITLE
Alexa notifications and Discord webhooks.

### PR DESCRIPTION
**Pull request by jusplainmike**
Added two services: discord, alexa-notify-me

Alexa notifications are providing using Notify-Me voice app. 
http://www.thomptronics.com/notify-me

Discord notifications using built in webhooks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,94 +33,94 @@ I've written an introductory post, explaining [what mqttwarn can be used for](ht
     + [Producing JSON](#producing-json)
   * [Notes](#notes)
   * [Press](#press)
-  
-      
+
```
Comment (rgitzel) [inline README.md]: Did you apply some sort of auto-formatting to this file?  There are a LOT of extraneous changes, making it very difficult to find any actual changes.  Please undo any unnecessary change.

File: services/gss2.py [hunk 2]
```diff
@@ -48,12 +48,12 @@ def plugin(srv, item):
         srv.logging.debug("Adding row to spreadsheet %s [%s]..." % (spreadsheet_url, worksheet_name))
         if os.path.isfile(oauth2_storage_filename):
             # Valid credentials from previously completed authentication?
-            srv.logging.debug(u"Trying to use credentials from file '%s'." % 
+            srv.logging.debug(u"Trying to use credentials from file '%s'." %
```
Comment (rgitzel) [inline services/gss2.py]: Are these actual changes?

Comment (jusplainmike) [reply services/gss2.py]: I opened the file using github desktop and atom and it auto removed the spaces in the file. See the missing spaces. Those were the only changes were deletion of extra spaces.

**Comment by jusplainmike** [comment]
Will fix spaces deletion issue.

**Comment by rgitzel** [comment]
@jusplainmike Cool, thanks! It's annoying how tools sometimes decide to do things all on their own. ;-)
